### ISSUE TITLE
cannot compute Minimum  - CategoricalCrossentropy

### ISSUE THREAD

**Issue by vtelenak** [SciSharp/TensorFlow.NET#901]
Hi

`
NDArray a = np.array(new float[,] { { 0, 1, 0 }, { 0, 0, 1 } });
NDArray b = np.array(new float[,] { { 0.05f, 0.95f, 0 }, { 0.1f, 0.8f, 0.1f } });
var ce = keras.losses.CategoricalCrossentropy();
Console.WriteLine(ce.Call(a, b));`

throw exception:

Tensorflow.InvalidArgumentError
  HResult=0x80131500
  Zpráva=cannot compute Minimum as input #1(zero-based) was expected to be a float tensor but is a double tensor
  Zdroj=TensorFlow.NET
  StackTrace:
   at Tensorflow.Status.Check(Boolean throwException)
   at Tensorflow.Eager.EagerRunner.TFE_FastPathExecute(FastPathOpExecInfo op_exec_info)
   at Tensorflow.Contexts.Context.ExecEagerAction(String OpType, String Name, ExecuteOpArgs args)
   at Tensorflow.Contexts.Context.ExecuteOp(String opType, String name, ExecuteOpArgs args)
   at Tensorflow.gen_math_ops.minimum[T1,T2](T1 x, T2 y, String name)
   at Tensorflow.math_ops.minimum[Tx,Ty](Tx x, Ty y, String name)
   at Tensorflow.clip_ops.<>c__DisplayClass0_0`2.<clip_by_value>b__0(NameScope <p0>)
   at Tensorflow.Binding.tf_with[TIn,TOut](TIn py, Func`2 action)
   at Tensorflow.clip_ops.clip_by_value[T1,T2](Tensor t, T1 clip_value_min, T2 clip_value_max, String name)
   at Tensorflow.Keras.BackendImpl.categorical_crossentropy(Tensor target, Tensor output, Boolean from_logits, Int32 axis)
   at Tensorflow.Keras.Losses.CategoricalCrossentropy.Apply(Tensor y_true, Tensor y_pred, Boolean from_logits, Int32 axis)
   at Tensorflow.Keras.Losses.Loss.Call(Tensor y_true, Tensor y_pred, Tensor sample_weight)
   at TensorflowTest.Program.Test() in C:\Users\\source\repos\TestKeras\TensorflowTest\Program.cs:line 249
   at TensorflowTest.Program.Main(String[] args) in C:\Users\\source\repos\TestKeras\TensorflowTest\Program.cs:line 28

Versions:
Tensorflow.NET 0.60.5
Tensorflow.Keras 0.6.5
SciSharp.Tensorflow.Redist 2.7.0

**Comment by vtelenak** [comment]
Great, it works :-)
